lib/MediaWiki/Bot.pm

```diff
@@ -2588,15 +2588,17 @@ sub db_to_domain {
     if (ref $wiki eq 'ARRAY') {
         my @return;
         foreach my $w (@$wiki) {
-            $wiki =~ s/_p$//;    # Strip off a _p suffix, if present
+            $wiki =~ s/_p$//;                               # Strip off a _p suffix, if present
```

Comment:
If you were trying to line that comment up with the one below, you're off by a bit ;)